src/components/Speciality/Intro.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react'
 import styled from 'styled-components'
+import Flex from 'styled-flex-component'
+import { Padding } from 'styled-components-spacing'
+import remcalc from 'remcalc'
+
 import { Row, Col, Grid } from '../grid'
 import { SectionTitle, CardTitle, Subtitle, BodyPrimary } from '../Typography'
-import { Padding } from 'styled-components-spacing'
 import BlueBackground from '../Common/BlueBackground'
-import Flex from 'styled-flex-component'
 
 const IntroBorder = styled(Col)`
   border: 1px solid rgba(255, 255, 255, 0.3);
 `
+
+const StyledBlueBackground = styled(BlueBackground)`
+  padding-top: ${remcalc(36)};
```

Comment:
I know it visually doesn't make a difference but perhaps we could use `120`? Just to be consistent with what we do [here](https://github.com/yldio/yld.io/pull/189/files#diff-a6f60c09f6002fca48bbddd4dc693548R25)?

Reply:
I don't understand what you mean by use `120` ? I agree that we may need to doc on occasion as this code is generally too fragile; and probably will not get any better when future devs will have to add more things on top of this.

Reply:
Ah it's not important. Let's leave it as is :+1: